test/unit/UnitTestPending.ts

```diff
@@ -129,7 +134,9 @@ describe('hasPending()', () => {
     const gasPrice = 1 * 1e12;
     const config = preConfig(mockConfig(), { client: '', gasPrice });
     const pending = await hasPending(config, mockTx({ address: startAddr, gasPrice }), {});
+    /* tslint:disable */
     expect(pending).to.be.false;
+    /* tslint:enable */
```

Comment:
Please add `/* tslint:disable:no-unused-expression */` to the top of the file instead of disabling ts-lint on separate lines.

Reply:
I would recommend to use different expression - method, instead of disabling tslint rule for whole file.

Reply:
That could work as well. `assert.isFalse()` maybe.